services/provider/proto/provider.proto

```diff
@@ -172,3 +189,50 @@ message ReportStatusResponse{
     // Contains subscription channel of provider operator for client operator to match
     string desiredClientOperatorChannel = 1;
 }
+
+// OnboardMirroringPeerRequest holds the required information to validate and onboard Storage Provider cluster
+message OnboardMirroringPeerRequest{
+    // onboardingTicket authenticates the secondary provider cluster
+    string onboardingTicket = 1;
```

Comment:
is this going to be the same ticket that we use today?

Reply:
Yes, this will be the same ticket but with a `role` for mirroring, changes for which are in https://github.com/red-hat-storage/ocs-operator/pull/2669